utils/tests_fetcher.py

```diff
@@ -396,16 +396,16 @@ def get_doctest_files(diff_with_last_commit: bool = False) -> List[str]:
             print(f"Parent commit: {commit}")
         test_files_to_run = get_diff_for_doctesting(repo, repo.head.commit, parent_commits)
 
-    # This is the full list of doctest tests
-    with open("utils/documentation_tests.txt") as fp:
-        documentation_tests = set(fp.read().strip().split("\n"))
+    # These are files not doctested yet.
+    with open("utils/not_doctested.txt") as fp:
+        not_doctested = set(fp.read().strip().split("\n"))
     # Do not run slow doctest tests
     with open("utils/slow_documentation_tests.txt") as fp:
         slow_documentation_tests = set(fp.read().strip().split("\n"))
 
     # So far we don't have 100% coverage for doctest. This line will be removed once we achieve 100%.
     test_files_to_run = [
-        x for x in test_files_to_run if x in documentation_tests and x not in slow_documentation_tests
+        x for x in test_files_to_run if x not in not_doctested and x not in slow_documentation_tests
```

Comment:
The files `slow_documentation_tests` won't be in `not_doctested` - they are doctested, but only on daily CI.

Reply:
once PR approved, I will remove the file `documentation_list.txt`